src/Symfony/Component/DependencyInjection/Tests/Loader/XmlFileLoaderTest.php

```diff
@@ -207,6 +208,20 @@ public function testParsesTags()
         }
     }
 
+    public function testParseTagsWithoutName()
+    {
+        try {
+            $container = new ContainerBuilder();
+            $loader = new XmlFileLoader($container, new FileLocator(self::$fixturesPath.'/xml'));
```

Comment:
this should not be inside the try...catch. or you could also use `@expectedException` and `@expectedExceptionMessage` directly